src/config_file.c

```diff
@@ -492,6 +493,47 @@ gboolean load_config(const gchar *filename, RaucConfig **config, GError **error)
 	return res;
 }
 
+// Something that is, or can be, mounted onto a mount point
+typedef struct {
+	gboolean is_device;  // vs being a loop mounted file
+	dev_t dev;  // the device, or for a file, the device the file is on
+	ino_t inode;  // inode of file for a non-device
+} MountableObj;
+
+// Take a device (or file) path and normalize it
+static MountableObj *normalize_mountable_object(const gchar *devicepath)
+{
+	MountableObj *obj;
+	GStatBuf st;
+
+	if (g_stat(devicepath, &st) == -1) {
+		/* Virtual filesystems like devpts trigger case */
+		g_debug("Can't stat '%s', must not be mountable", devicepath);
```

Comment:
I can't get the meaning of the error message, what do you mean by `must not be mountable`?.

And it might also be useful that we evaluate errno here to complete information using `gstrerror(errno)`.

Reply:
I can't think of anything which can not be stat'ed, but can be mounted.  So if the purpose is to find a mounted device or file, and the path can not be stated, then it must not be a mounted device or file.

Reply:
I stick to my opinion that the message is misleading because one can read 'must not be mountable' as an instruction in the sense of 'It should not be a mountable path', which is quite the opposite of what you intend to say here.
What about something like: `Not a mountable path: %s: %s", devicepath, gstrerror()`

What `gstrerror(errno)` will tell you in most cases is something like 'No such file or directory', which is quite more concrete than saying that it is not mountable. If we have a concrete meaningful error message: why hiding it?